src/compiler/csharp_plugin.cc

```diff
@@ -62,14 +63,17 @@ class CSharpGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
         // The option may be removed or file names generated may change
         // in the future.
         base_namespace = options[i].second;
+      } else if (options[i].first == "enable_nrt") {
```

Comment:
Also, I'd say let's add a note saying that the new "enable_nrt" (or whatever name we end up choosing) option is experimental (same as "base_namespace")